client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-list.js

```diff
@@ -106,7 +107,9 @@ const ShippingZoneLocationList = ( { siteId, loaded, translate, locations, actio
 			<ExtendedHeader
 				label={ translate( 'Zone locations' ) }
 				description={ translate( 'Define the places that are included in this zone.' ) } >
-				<Button onClick={ onAddLocation } disabled={ ! loaded } >{ translate( 'Edit locations' ) }</Button>
+				<Button onClick={ onAddLocation } disabled={ ! loaded } >
+					{ isEmpty( locations ) ? translate( 'Add locations' ) : translate( 'Edit locations' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Location' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).